scripts/superdesk-archive/views/metadata-view.html

```diff
@@ -47,11 +47,15 @@
         </dl>
         <dl>
             <dt translate>ANPA Category</dt>
-            <dd ng-if="item['anpa_category']">{{ item['anpa_category'] | mergeWords:'name' }}</dd>
+            <dd ng-if="item.anpa_category">{{ item.anpa_category | mergeWords:'name' }}</dd>
         </dl>
-        <dl >
+        <dl>
             <dt translate>Subject</dt>
-            <dd ng-if="item.subject && item.subject.length > 0">{{ item.subject | mergeWords:'name'}}</dd>
+            <dd>{{ item.subject | mergeWords:'name':'subject_custom'}}</dd>
+        </dl>
+        <dl>
+            <dt translate>Category</dt>
```

Comment:
Category is a custom cv for NTB, so it should not appear here. The solution is to read the list of cv for current item's profile and show them dynamically in the way that are shown on item header.